src/FeedIo/FeedInterface.php

```diff
@@ -56,4 +56,6 @@ public function add(ItemInterface $item) : FeedInterface;
      * @return ItemInterface
      */
     public function newItem() : ItemInterface;
+
+    public function addNS($ns, $dtd) : FeedInterface;
```

Comment:
Is it possible to add parameters' types ? If I get it well, they both can be typed as `string`